internal/host/transition_test.go

```diff
@@ -1631,7 +1635,7 @@ var _ = Describe("Refresh Host", func() {
 				dstState:           models.HostStatusInsufficient,
 				machineNetworkCidr: "1.2.3.0/24",
 				role:               models.HostRoleMaster,
-				statusInfoChecker:  makeValueChecker(statusInfoNotReadyForInstall),
+				statusInfoChecker:  makeValueChecker(formatStatusInfoNotReadyForInstall("Require at least 4 CPU cores for master role, found only 2, Require at least 16 GiB RAM role master, found only 8")),
```

Comment:
Seems that when there are multiple messages each containing a comma "," then it is hard to distinguish which is the message

Reply:
Agree.  Maybe join with semi-colon?